e2e/app/aou-elements/xpath-defaults.ts

```diff
@@ -0,0 +1,128 @@
+import TextOptions from './TextOptions';
```

Comment:
best practice: standardized and centralize xpaths selectors in `xpath-defaults.ts`.